baton-maven-plugin/src/main/java/org/technologybrewery/baton/util/CommonUtils.java

```diff
@@ -0,0 +1,24 @@
+package org.technologybrewery.baton.util;
+import com.vdurmont.semver4j.Semver;
+
+/**
+ * Common logic used when performing a migration
+ */
+public final class CommonUtils {
+
+
+
+    /**
+     * Checks if version1 is less than version2 using the Semver4j library.
+     *
+     * @param version1
+     * @param version2
+     * @return isLessThanVersion - if true, version1 is less than version2.
+     */
+    private CommonUtils(){
```

Comment:
A: seems like the private constructor got injected before the method.  There is some extra whitespace above too